vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/test/java/com/vaadin/flow/component/combobox/ComboBoxBaseTest.java

```diff
@@ -274,7 +274,7 @@ public Stream<String> fetch(Query query) {
                     }
                 });
 
-        comboBox.setDataProvider(dataProvider, filter -> null);
+        comboBox.setItems((DataProvider) dataProvider);
```

Comment:
I wonder if it would be better to make `dataProvider` of type `DataProvider<String, String>` and avoid the unchecked cast

Reply:
Good point.